rules/windows/execution_windows_powershell_susp_args.toml

```diff
@@ -2,7 +2,7 @@
 creation_date = "2024/09/06"
 integration = ["windows", "system", "sentinel_one_cloud_funnel", "m365_defender", "crowdstrike"]
```

Comment:
```suggestion
integration = ["endpoint", "windows", "system", "sentinel_one_cloud_funnel", "m365_defender", "crowdstrike"]
```

Reply:
@Samirbous decided to not include it because we have a very similar rule for endpoint, should I add it?